CHANGELOG.md

```diff
@@ -58,6 +58,10 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 * **The `type_list` header has moved:**
   If you included `<seqan3/core/type_list.hpp>` you need to change the path to `<seqan3/core/type_list/type_list.hpp>`.
 
+#### I/O
+
+* The `field::FLAG` of SAM/BAM input and output is not an enum instead of a simple integer (see seqan3::sam_flag).
```

Comment:
```suggestion
* The `field::FLAG` of SAM/BAM input and output is now an enum instead of a simple integer (see seqan3::sam_flag).
```
This is an API change. Can you please document how this will affect people and how to fix possible compiler errors.

Reply:
I don't really know what to say here...? 
You cannot use integers anymore when writing but you need to static_cast your integers into the respective enum value?
And when you read, well you need to change all your types from integer to sam_flags wherever you use them..

Reply:
@rrahn ?